python3/bin/xe-reset-networking

```diff
@@ -24,7 +24,7 @@ pool_conf = '@ETCXENDIR@/pool.conf'
 inventory_file = '@INVENTORY@'
 management_conf = '/etc/firstboot.d/data/management.conf'
 network_reset = '/var/tmp/network-reset'
-
+rename_script_exists = os.path.exists("/etc/sysconfig/network-scripts/interface-rename.py")
```

Comment:
to avoid typos it'd be good to make this long string here a constant, so that testing for its existence and running it is done on the exact same string, e.g. RENAME_SCRIPT=